backend/src/routers/gameRouter.ts

```diff
@@ -268,13 +268,17 @@ gameRouter.post('/filter', validateRequestBody(filterGamesSchema), async (req, r
  *
  * @apiSuccessExample Success-Response:
  * HTTP/1.1 200 OK
- *  [
  *   {
- *    "id": "clgkri8kk0000przwvkvbyj95",
+ *    "message": "Game removed"
  *   }
- * ]
  *
  * @apiUse ZodError
+ *
+ * @apiError (400) {object} Unauthorized Must be logged in to add game
```

Comment:
Ignoring that we say add game. This is not true, we don't check auth when removing a game currently